torch/_dynamo/variables/builder.py

```diff
@@ -1512,6 +1512,8 @@ def _clone_input(value):
         torch._functorch.vmap._validate_and_get_batch_size,
         torch._C._functorch._grad_increment_nesting,
         torch._C._functorch._grad_decrement_nesting,
+        torch._C._functorch.maybe_current_level,
+        torch._C._is_fwd_grad_enabled,
```

Comment:
> torch._C._is_fwd_grad_enabled

We don't need that for this PR, right? I'm not sure it should go into the graph (it's not a Tensor operation and it's not something we need in the graph -- Dynamo should probably just guard on this)

> torch._C._functorch.maybe_current_level

This also probably shouldn't go into the graph -- Dynamo already guards against this, right? (via the functorch_stack_match guard).